source/core/utils/dns-ip-version.ts

```diff
@@ -0,0 +1,24 @@
+export type DnsIpVersion = 'auto' | 'ipv4' | 'ipv6';
+export type DnsIpFamily = 0 | 4 | 6;
```

Comment:
Why are you exporting this? It's only used here.

Reply:
That was a mistake, I used to use it, but when I removed the reference I forgot the export.